.github/workflows/tests.yml

```diff
@@ -47,3 +47,10 @@ jobs:
         run: |
           pip install -r requirements-channels.txt
           pip install .
+
+      - name: Run Tests
```

Comment:
@zerolab we didn't run tests before with channels enabled before. Any reason for that?
If so, this would be removed.

Reply:
it was a glaring omission 🤦‍♂️

Reply:
ok!